Makefile

```diff
@@ -377,6 +377,7 @@ release-qa: require-version require-s3cmd
 		VERSIONED=lantern-installer-$$VERSION$$SUFFIX && \
 		echo "Copying $$VERSIONED" && \
 		$(S3CMD) cp s3://$(S3_BUCKET)/$$NAME s3://$(S3_BUCKET)/$$VERSIONED; \
```

Comment:
I think this would be better if chained with `&&` to the next command:

```
 		$(S3CMD) cp s3://$(S3_BUCKET)/$$NAME s3://$(S3_BUCKET)/$$VERSIONED && \

```